src/www/ui/api/Controllers/GroupController.php

```diff
@@ -244,4 +244,58 @@ public function getGroupMembers($request, $response, $args)
     }
     return $response->withJson($res->getArray(), $res->getCode());
   }
+
+
+  /**
+   * Add a user to a group
+   *
+   * @param ServerRequestInterface $request
+   * @param ResponseHelper $response
+   * @param array $args
+   * @return ResponseHelper
+   */
+  public function addMember($request, $response, $args)
+  {
+    $returnVal = null;
+    $dbManager = $this->dbHelper->getDbManager();
+
+    $body = $this->getParsedBody($request);
+
+    $group_pk = intval($args['id']);
+    $newuser = intval($body['userId']);
+    $newperm = intval($body['perm']);
+
+    if (!isset($newperm)) {
+      $returnVal = new Info(400, "ERROR - no default permission provided", InfoType::ERROR);
+    } else if (!$this->dbHelper->doesIdExist("groups", "group_pk", $group_pk)) {
+      $returnVal = new Info(404, "Group id not found!", InfoType::ERROR);
+    } else if (!$this->dbHelper->doesIdExist("users", "user_pk", $newuser)) {
+      $returnVal = new Info(404, "User id not found ! ".$newuser, InfoType::ERROR);
+    } else if ($newperm < 0 || $newperm > 2) {
+      $returnVal = new Info(400, "ERROR - Permission should be in range [0-2]", InfoType::ERROR);
+    } else {
+      try {
+        $dbManager->prepare($stmt = __METHOD__ . ".getByGroupAndUser",
+          "SELECT group_user_member_pk FROM group_user_member WHERE group_fk=$1 AND user_fk=$2");
+        $fetchResult = $dbManager->execute($stmt, array($group_pk, $newuser));
+        $fetchResult = $dbManager->fetchAll($fetchResult);
+        $dbManager->freeResult($fetchResult);
+
+        // Do not produce duplicate
+        if (empty($fetchResult)) {
+          $dbManager->prepare($stmt = __METHOD__ . ".insertGUP",
+            "insert into group_user_member (group_fk, user_fk, group_perm) values ($1,$2,$3)");
```

Comment:
Please following coding style guidelines:
```suggestion
            "INSERT INTO group_user_member (group_fk, user_fk, group_perm) VALUES ($1, $2, $3)");
```